src/panels/lovelace/cards/hui-entities-card.js

```diff
@@ -116,28 +102,18 @@ class HuiEntitiesCard extends EventsMixin(PolymerElement) {
     this._elements = [];
 
     for (const entity of entities) {
-      const entityId = entity.entity;
-
       const element = createEntityRowElement(entity, this.hass);
-      if (entityId && !DOMAINS_HIDE_MORE_INFO.includes(computeDomain(entityId))) {
-        element.classList.add('state-card-dialog');
-        element.addEventListener('click', () => this.fire('hass-more-info', { entityId }));
```

Comment:
We should keep this here. We don't know where EntityRow will be used.

Reply:
Although, I do really like if we only attach click listener to the left part of the generic row. Interacting with controls should never open more info.